client/src/components/Message.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export interface MessageProps {
+  id: number;
```

Comment:
@YaelEdri - The id prop is not relevant you can pass it anyway because it's part of react component props. just remove it.

Reply:
Yes, i forgot to delete it when I finished with it...